packages/pluggableWidgets/line-chart-native/src/components/Legend.tsx

```diff
@@ -0,0 +1,44 @@
+import { createElement, ReactElement, useMemo } from "react";
+import { Text, View } from "react-native";
+
+import { LineChartStyle } from "../ui/Styles";
+
+export interface LegendProps {
+    series: LegendSeries[];
+    style: LineChartStyle;
+}
+
+export interface LegendSeries {
+    name?: string;
+    stylePropertyName?: string;
+}
+
+export function Legend(props: LegendProps): ReactElement | null {
+    const { series, style } = props;
+
+    const legendItems = useMemo(
+        () =>
+            series.flatMap((series, index) => {
```

Comment:
The flatmap here is actually not necessary as ReactElement[] is a valid JSX

Reply:
Converted `flatMap` into `map`